packages/env/README.md

```diff
@@ -92,3 +194,5 @@ Positionals:
 The file should be located in the same directory from which you run `wp-env` commands for a project. So if you are running `wp-env` in the root directory of a plugin, `wp-env.json` should also be located there. 
 
 Each item in the `themes` or `plugins` array should be an absolute or relative path to the root of a different theme or plugin directory. Relative paths will be resolved from the current working directory, which means they will be resolved from the location of the `wp-env.json` file.
+
+<br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
```

Comment:
Why is this here? Also, the `align` attribute is not valid HTML5.

Reply:
> Why is this here? Also, the `align` attribute is not valid HTML5.

These have been included in all of the package READMEs, so I assume it was added for consistency's sake.

Whether it makes sense to include this image... I don't see a ton of value in it it, but I grant it's likely intended more as personality / "flair" to associate with WordPress.

Re: `align`, I think it's out of necessity for it to appear centered in GitHub. I've seen (and used) it in Markdown files in other projects. I don't know that there's an alternative that works equally well for GitHub Markdown file previews.